mls-rs/src/group/commit.rs

```diff
@@ -523,6 +537,12 @@ where
         #[cfg(not(feature = "std"))]
         let time = None;
 
+        let time = if commit_time.is_some() {
```

Comment:
Here is another place that can be simplified by requiring a `MlsTime` instead of `Option<MlsTime>` above.